### PR TITLE
Update first-move-module.md

### PR DESCRIPTION
**Pull request by Fei94043**
### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3184)

### CODE REVIEW

**Review by davidiw — COMMENTED** [review]
this is great, as you're probably aware, we're rewriting the tutorials, so I'll keep this in mind. One nit, you include your user paths, whereas using relative paths would be better

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ For this tutorial, will be focusing on `hello_blockchain/src` and re-using the `
 
 You can find the rust project [here](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site/static/examples/rust)
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/tutorials/first-move-module.md:39]: rust → Rust

Comment (rajkaramchedu) [reply developer-docs-site/docs/tutorials/first-move-module.md:39]: Put a period at the end of either "here" as `[here.]` or after the closing bracket of the link.

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ For this tutorial, will be focusing on `hello_blockchain/src` and re-using the `
...
 
+The key is that you need have two terminals open at the same time:  One is Move terminal, the other is Python terminal, and the sequence of events of switching these two terminals is very important.
+
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/tutorials/first-move-module.md:42]: I suggest we remove the above passage. We should rely on the intelligence of the reader to figure this out themselves. Or else we will have to tell them a whole lot more low-level things to be consistent in our docs.

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 3]
```diff
@@ -259,38 +262,51 @@ For Rust:
...
 
-- After a few moments it will mention that "Update the module with Alice's address, build, copy to the provided path,
-  and press enter."
-- In the "Move Window" terminal, and for the Move file we had previously looked at:
-  - Copy Alice's address
-  - Compile the modules with Alice's address by `aptos move compile --package-dir . --named-addresses HelloBlockchain=0x{alice_address_here}`. Here, we replace the generic named address `HelloBlockChain='_'` in `hello_blockchain/move.toml` with Alice's Address
-  - Copy `build/Examples/bytecode_modules/Message.mv` to the same folder as this tutorial project code
-- Return to your other terminal window, and press "enter" at the prompt to continue executing the rest of the code
+On Python terminal:
+ python git:(c6841bd6b0) python3 hello_blockchain.py Message.mv
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/tutorials/first-move-module.md:266]: What is this `python git:(c6841bd6b0)`? Maybe this is part of the command line prompt on your local terminal? Remove this please. The correct command is `python3 hello_blockchain.py Message.mv`

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 4]
```diff
@@ -259,38 +262,51 @@ For Rust:
...
+Bob: {'type': '0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>', 'data': {'coin': {'value': '5000'}, 'deposit_events': {'counter': '1', 'guid': {'id': {'addr': '0x43f9269af65f97e24d71d874de7e070ad889c7da7254d14307e7c887b4e4ee7c', 'creation_num': '1'}}}, 'withdraw_events': {'counter': '0', 'guid': {'id': {'addr': '0x43f9269af65f97e24d71d874de7e070ad889c7da7254d14307e7c887b4e4ee7c', 'creation_num': '2'}}}}}
+
+Update the module with Alice's address, build, copy to the provided path, and press enter.  
+
+
+AT the point, move to the other Move terminal: 
+
+On the Move terminal
+/Users/feizheng/aptos-core/aptos-move/move-examples/hello_blockchain
+➜  hello_blockchain git:(c6841bd6b0) aptos move compile --package-dir . --named-addresses HelloBlockchain=0x10bf3a5f10a4dcd6b387221559ca4cf6ead4850ae3499816494fd94845938d7b
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/tutorials/first-move-module.md:283]: Remove `hello_blockchain git:(c6841bd6b0)`

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 5]
```diff
@@ -259,38 +262,51 @@ For Rust:
...
+{
+  "Result": [
+    "10BF3A5F10A4DCD6B387221559CA4CF6EAD4850AE3499816494FD94845938D7B::message"
+  ]
+}
+
+➜  bytecode_modules git:(c6841bd6b0) cp message.mv /Users/feizheng/aptos-core/developer-docs-site/static/examples/python/
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/tutorials/first-move-module.md:290]: Remove `bytecode_modules git:(c6841bd6b0)`

**Review by rajkaramchedu — CHANGES_REQUESTED** [review]

**Comment by davidiw** [comment]
Please review the updated doc. I think much of the feedback has been integrated. Agreed with @rajkaramchedu  we cannot ... simplify it too much.

**Pull request closed without merging**